### PR TITLE
feat(icons): add booster and voicemail 

### LINKED ISSUE #291: 2 new icons to be added to the DNA library
[2 new icons for yousee.zip](https://github.com/youseedk/dna/files/6375289/2.new.icons.for.yousee.zip)
Booster.svg
Voicemail.svg
Please add these icons to the DNA

### PR DESCRIPTION
**Pull request by laukondrup**
- Closes #291

### CODE REVIEW

File: src/assets/svg/booster.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg width="42" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3 26C3 24.8954 3.89543 24 5 24H37C38.1046 24 39 24.8954 39 26V36.2756C39 37.3802 38.1046 38.2756 37 38.2756H5C3.89543 38.2756 3 37.3802 3 36.2756V26Z" stroke="#191919" stroke-width="2"/>
+<path d="M8 29H12" stroke="#191919" stroke-width="2" stroke-linecap="round"/>
```
Comment (razvandk) [inline src/assets/svg/booster.svg:3]: i think for all strokes should be `stroke="currentColor"` not `stroke="#191919"`

**Review by razvandk — CHANGES_REQUESTED** [review]

File: src/assets/svg/booster.svg (hunk 1 above)
Comment (laukondrup) [reply src/assets/svg/booster.svg]: ahh, of course, fixed!

File: src/assets/svg/booster.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<svg width="42" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Lubbe91) [inline src/assets/svg/booster.svg:1]: I think it's suppose to be 40
```suggestion
<svg width="40" height="40" viewBox="0 0 42 40" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment (razvandk) [reply src/assets/svg/booster.svg:1]: they should be `40 x 40` but we need to double check that the icon its still centrated

Comment (laukondrup) [reply src/assets/svg/booster.svg:1]: the icon is being cut off 
image

Comment (laukondrup) [reply src/assets/svg/booster.svg]: fixed

File: src/assets/svg/booster.svg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (grandorf) [inline src/assets/svg/booster.svg:1]: You should properly remove the width and height attributes so the parent element controls the size of the icon. This is how all other icons are implemented. (required)

File: src/assets/svg/booster.svg [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 26C2 24.8954 2.89543 24 4 24H36C37.1046 24 38 24.8954 38 26V36.2756C38 37.3802 37.1046 38.2756 36 38.2756H4C2.89543 38.2756 2 37.3802 2 36.2756V26Z" stroke="currentColor" stroke-width="2"/>
```
Comment (grandorf) [inline src/assets/svg/booster.svg:2]: Missing indentation/formatting (optional)

**Review by grandorf — CHANGES_REQUESTED** [review]
Please target the develop branch. We/I usually added more fixed issues to this branch before releasing. Otherwise it will be many releases. In the end before merging to master please bump npm up with a minor version.
Then after master is pushed you should do `git push --tags` this will create a new release and deploy to Npm.

File: src/assets/svg/booster.svg (hunk 3 above)
Comment (laukondrup) [reply src/assets/svg/booster.svg]: 👍

File: src/assets/svg/booster.svg (hunk 4 above)
Comment (laukondrup) [reply src/assets/svg/booster.svg]: 👍

**Pull request merged**